chia/wallet/wallet_transaction_store.py

```diff
@@ -79,6 +91,8 @@ async def create(cls, db_wrapper: DBWrapper):
         self.tx_record_cache = {}
         self.tx_submitted = {}
         self.unconfirmed_for_wallet = {}
+        self.last_wallet_tx_resend_time = int(time.time())
+        self.wallet_tx_resend_timeout_secs = 60 * 60
```

Comment:
Is this no longer used, I couldn't find any other reference to it in WalletTransactionStore

Reply:
That's right - I moved this out of WalletTransactionStore. I thought it made it more clear, and better separated the job of WalletTransactionStore. Thank you.